src/config/graphical-session/graphics-drivers/nvidia.md

```diff
@@ -0,0 +1,64 @@
+# Nvidia GPU
+
+## nouveau (Open Source drivers)
+
+Install the Mesa DRI drivers and Xorg Drivers:
+
+```
+# xbps-install -S mesa-nouveau-dri xf86-video-nouveau
+```
+
+> Note: This is already included in the `xorg` metapackage, but it is needed
+> when installing xorg via `xorg-minimal` or for running a Wayland compositor.
+
+In order to run 32-bit programs, install the `mesa-nouveau-dri-32bit` package.
+
+These drivers are developed mostly by the community, with little input from
+Nvidia, and are not as performant as the proprietary drivers. They are required
+in order to run Wayland compositors based on `wlroots`.
+
+## nvidia (Proprietary drivers)
+
+If your graphics card is fairly new, install the drivers:
+
+```
+# xbps-install -S nvidia
+```
+
+If you don't already know what graphics card you have, run:
+
+```
+# lspci -k | grep -A 2 -E "(VGA|3D)"
+```
+
+Check if your graphics card belongs to the legacy branch. If you find it on
+[this](https://www.nvidia.com/en-us/drivers/unix/legacy-gpu/) list, you should
+install the appropriate driver.
```

Comment:
You can probably write "the appropriate driver, either `nvidia390` or `nvidia340`". The version is already in the name.